From dcc2a4344b064ab91a0a89baa2453244de78ceb4 Mon Sep 17 00:00:00 2001
From: Mei Liu <liumbj@linux.vnet.ibm.com>
Date: Tue, 10 Nov 2015 12:34:24 +0800
Subject: [PATCH] Add awk and getopt as dependencies of znetconf

znetconf command is part of s390utils-base package. It depends on
awk and getopt.

This patch is used to fix the following error:

dracut:/#
znetconf -c

/usr/sbin/znetconf: line 70: awk: command not found
/usr/sbin/znetconf: line 1138: getopt: command not found

Signed-off-by: Mei Liu <liumbj@linux.vnet.ibm.com>
---
 modules.d/80cms/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/80cms/module-setup.sh b/modules.d/80cms/module-setup.sh
index 1afac16..05c93e5 100755
--- a/modules.d/80cms/module-setup.sh
+++ b/modules.d/80cms/module-setup.sh
@@ -27,7 +27,7 @@ install() {
     inst_script "$moddir/cmsifup.sh" /sbin/cmsifup
     inst_multiple /etc/cmsfs-fuse/filetypes.conf /etc/udev/rules.d/99-fuse.rules /etc/fuse.conf \
         cmsfs-fuse fusermount ulockmgr_server bash insmod rmmod cat normalize_dasd_arg sed \
-        $(rpm -ql s390utils-base)
+        $(rpm -ql s390utils-base) awk getopt
 
     inst_libdir_file "gconv/*"
     #inst /usr/lib/locale/locale-archive
